validator/validator/validator.py

```diff
@@ -486,7 +492,7 @@ def test_next_miner(self):
         self.contest_state.miner_score_versions[uid] = WEIGHTS_VERSION
 
     def get_score_buckets(self) -> list[WinnerList]:
-        sorted_contestants = self.metrics.get_sorted_contestants()
+        sorted_contestants = cast(list[tuple[Uid, float]], self.metrics.get_sorted_contestants())
```

Comment:
Why the cast?

Reply:
pycharm complains without it
<img width="692" alt="Screenshot 2024-09-04 at 11 11 23 AM" src="https://github.com/user-attachments/assets/01b3c73f-1a33-42c3-a26e-901152278e4c">

Reply:
The type-hint for get_sorted_contestants should use Uid instead of int